hangman/word_generator.py

```diff
@@ -0,0 +1,17 @@
+from requests import get
+import random
+from typing import List
+
+
+class WordGenerator:
+    def __init__(self, url) -> None:
+        self.api: str = url
+        self.words: List[str] = []
```

Comment:
also you are maintaining a list of these words in memory, for now this will work, what if we had to keep million words, how would you make changes to your design then /?